src/libraries/System.Security.Cryptography/tests/MLDsaOpenSslTests.Unix.cs

```diff
@@ -116,11 +116,11 @@ public void MLDsaOpenSsl_DuplicateKeyHandleLifetime()
 
         private static void VerifyInstanceIsUsable(MLDsaOpenSsl mldsa)
         {
-            byte[] seed = new byte[mldsa.Algorithm.PrivateSeedSizeInBytes];
-            Assert.Equal(mldsa.Algorithm.PrivateSeedSizeInBytes, mldsa.ExportMLDsaPrivateSeed(seed)); // does not throw
+            byte[] seed = mldsa.ExportMLDsaPrivateSeed();
+            Assert.NotEqual(0, seed.Length); // does not throw
```

Comment:
Actually, the "does not throw" was just saying "the instance didn't throw while exporting", so the two lines can probably be changed to something like

```C#
mldsa.ExportMLDsaPrivateSeed(); // does not throw
```

or, Jeremy-style

```C#
// Assert.NoThrow
mldsa.ExportPrivateSeed();
// Assert.NoThrow
mldsa.ExportMLDsaSecretKey();

// usable
byte data = [ 1, 2, 3 ];
...
```

Reply:
Resolving since I went with the other comment